src/messages/MNVMeofGwBeacon.h

```diff
@@ -0,0 +1,126 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+/*
+ * Ceph - scalable distributed file system
+ *
+ * Copyright (C) 2023
```

Comment:
```suggestion
 * Copyright (C) 2023 IBM, Inc.
```

here and everywhere else